.github/renovate.json5

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
Note that I removed references to a dependency dashboard. I don't remember personally experiencing benefit from it, and have always just gone with issues. Removed from this initial proposal to reduce noise & impact of PR. But it's not a hard opinion of mine. 🙂

Reply:
I love the Dependency Dashboard! ❤️ Here's why:

- You may use the [Dependency Dashboard approval workflow](https://docs.renovatebot.com/key-concepts/dashboard/#dependency-dashboard-approval-workflow) to approve certain updates before you get them. So you _first_ select the checkbox in the Issue, then you get the Renovate PR. Now you only get "noise" when you want to, for those updates.
- You can see any updates that you've ignored, for whatever reason, so you don't forget about them.

I usually pin the Dependency Dashboard issue to the top of the Issues list. That way I always know where to find it, no matter what repository I'm in.